ui-design-lib/src/main/java/co/electriccoin/zcash/ui/design/component/Button.kt

```diff
@@ -40,24 +50,30 @@ fun PrimaryButton(
         horizontal = ZcashTheme.dimens.spacingDefault,
         vertical = ZcashTheme.dimens.spacingSmall
     ),
-    enabled: Boolean = true
+    enabled: Boolean = true,
+    buttonColor: Color = MaterialTheme.colorScheme.primary,
+    textColor: Color = MaterialTheme.colorScheme.onPrimary,
 ) {
     Button(
         shape = RectangleShape,
+        enabled= enabled,
+        modifier = modifier
+            .padding(outerPaddingValues)
+            .shadow(
+                Color.Black,
```

Comment:
These color definitions could be part of the `Colors` file. We could then access them with `ZcashTheme.colors`.